src/languages/en.js

```diff
@@ -9,7 +9,7 @@ export default {
         to: 'To',
         optional: 'Optional',
         split: 'Split',
-        request: 'Request',
+        request: ({amount}) => `Request ${amount}`,
```

Comment:
Given that this is `Request ${amount}` should we move this outside `common`?

Reply:
> Given that this is `Request ${amount}` should we move this outside `common`?

Moved to 'iou'